index.html

```diff
@@ -2,17 +2,212 @@
-    <title>Hello world</title>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="./css/style.css">
+  <script src="https://kit.fontawesome.com/1ada02b4f0.js" crossorigin="anonymous"></script>
+  <title>LEEHAR'S FASHION WEEK </title>
 </head>
 
 <body>
-    <header>
-        <h1 class="header">Hello world!</h1>
-    </header>
+  <header>
+    <div class="top-navbar">
+      <ul>
+        <li>
+          <a href="#"><i class="fa fa-twitter"></i></a>
+        </li>
+        <li>
+          <a href="#"><i class="fa fa-facebook"></i></a>
+        </li>
+        <li><a href="#">Celebrities</a></li>
+        <li><a href="#">Vlog</a></li>
+        <li><a href="#"> Logout</a></li>
+      </ul>
+    </div>
+    <div class="main-navbar flex-center">
+      <div class="logo">
+        <a href="#"><img src="./assets/creative-tailor.png" alt="fashion logo" /></a>
+      </div>
+      <nav class="navbar">
+        <div class="hamburger">
+          <img src="./assets/hamburger.svg" alt="Menu Hamburger" />
+        </div>
+        <ul class="nav-links">
+          <li class="close-menu-btn">
+            <button type="submit">
+              <span>&times;</span>
+            </button>
+          </li>
+          <li><a href="index.html" id="current">Home</a></li>
+          <li><a href="about.html">About</a></li>
+          <li><a href="#">Models</a></li>
+          <li><a href="#">Tickets</a></li>
+          <li id="leehar-line-lg" class="flex-center"><a href="#">LEEHAR'S LINE</a></li>
+        </ul>
+      </nav>
+    </div>
+  </header>
+  <section class="home-section">
+    <div class="home-wrapper">
+      <div class="home-section-header">
+        <h2 class="home-message">"Hello! Fashionista"</h2>
+        <h1 class="home-info">
+          It's Leehar's fashion week🎉👙👗 where: Fashion is like eating, you shouldn't stick to the same menu😉👌
+        </h1>
+      </div>
+      <div class="home-text flex-center">
+        <p>
+          Create your own visual style... let it be unique for yourself and yet identifiable for others because Style is
+          a reflection of your attitude and your personality.
+        </p>
+      </div>
+      <div class="opening-hours">
+        <p>2021.12.17(FRI) ~ 19(SUN)</p>
+      </div>
+      <div class="home-text-2">
+        <p>
+          @ EKO Hotel and suites, Oriental Hotel and more.
+        </p>
+      </div>
+    </div>
+  </section>
 
+  <section class="main-program">
+    <div class="main-program-container flex-center">
+      <h3 class="main-program-header">Main Events</h3>
+      <p class="main-program-line"></p>
+      <div class="main-grid">
+        <div class="main-program-content flex-center">
+          <div class="main-program-img">
+            <img src="https://img.icons8.com/doodle/50/000000/red-carpet.png" alt="red carpet" />
```

Comment:
- [ ] This whole section overflows outside the container in small mobile screens from 320px and partial overflow from 375px 
![image](https://user-images.githubusercontent.com/33067256/128326770-5dd7cf3a-ebda-45f0-8b64-af3f849815e6.png)
![image](https://user-images.githubusercontent.com/33067256/128326867-b500fa5f-50a6-444b-a5ba-6661ffeae833.png)

Reply:
This has been worked on